_posts/2015-09-18-epages-you-2015.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Annual company event: ePagees rock the YOU"
+date: "2015-09-18 07:21:00"
+icon: users
+categories: events
+authors: ["Birgit"]
+---
+
+Creative team presentations, team building activities, workshops and lot's of fun: last weeks' activities and corridor talks at ePages all revolved around the ePages annual company event: the YOU.
+The YOU is a formal, annual company event which all ePages employees are required to attend.
+This sounds really tough.
+But it is not.
+It is incredibly fun, although it is hard work, too and involves long hours.
```

Comment:
Over the two day's the YOU requires hard work and long hours, but is an incredibly fun event.